### PR TITLE
Removes unused templates and CSS

### PR DESCRIPTION
**Pull request by anselmbradford**
## Removals

- Templates from `_includes`:
  - `leadership-calendar-table.html`
  - `macros/mobile-carousel.html`
  - `post-macros.html`
  - `posts-paginated.html`
  - `templates/office-contact.html`

- CSS:
  - `bureau-bio`, which was only used in markup.
  - `post-preview`, and `post-pagination`, `post_meta `, `post_byline`, `post_featured-img`, `summary`, which were unused outside of removed templates.
  - `featured-topic`, `expandable__filters`, `float-media`, `rwd-crop`, `video-poster`, `media_body__event`, `popular-stories`, `related-posts`, `post-summary-image`, `share__horizontal`, `share__align-right`, which only appeared in CSS, not markup.
  - `share-icon`, which was superseded by the social media molecule.
  -  `comments` and `comments_policy`, which appeared to be unused disqus CSS classes. 

- Docs:
  -  Outdated repeated content dev tip for sheer.
  - Doc code example with not applicable todo comment.

## Testing

1. `./setup.sh` should run and site should function as before.

### CODE REVIEW

File: cfgov/unprocessed/css/blog-docs.less [hunk 1]
```diff
@@ -5,207 +5,29 @@
...
-        - "Official icons are still being developed, hence the ?."
+    name: Press Information
+    markup: |
+      <div class="press-information">
+          <h1 class="header-slug">
+              <span class="header-slug_inner">
+                  Press information
+              </span>
+          </h1>
+          <p class="short-desc">
```
Comment (jimmynotjim) [inline cfgov/unprocessed/css/blog-docs.less]: `short-desc` doesn't exist any more. Should this block be updated?

File: cfgov/unprocessed/css/blog-docs.less [hunk 2]
```diff
@@ -5,207 +5,29 @@
...
+              If you want to repbulish the article or have questions about the content,
+              please contact the press office.
+          </p>
+          <p>
+              <a class="sans"
```
Comment (jimmynotjim) [inline cfgov/unprocessed/css/blog-docs.less]: `sans` also doesn't exist any more.

File: cfgov/unprocessed/css/post.less [hunk 3]
```diff
@@ -384,31 +377,17 @@
...
-        margin:
-            unit(30px / @base-font-size-px, em)
-            0
-            unit(20px / @base-font-size-px, em);
-    });
-}
+// TODO: `post_body` is used on article.html and events/_event-detail.html.
+//       Refactor to use CF classes if possible.
 
 .post_body {
```
Comment (jimmynotjim) [inline cfgov/unprocessed/css/post.less]: Pretty sure this one can go away completely. @Scotchester and @nataliafitzgerald worked out all the text combos in CF a long time ago.

Comment (anselmbradford) [inline cfgov/unprocessed/css/post.less]: It's currently adding some margin on the individual block pages. Definitely can be refactored, but for this PR I'm trying to remove things not refactor them.

File: cfgov/unprocessed/css/blog-docs.less (hunk 1 above)
Comment (anselmbradford) [inline cfgov/unprocessed/css/blog-docs.less]: Fixed in https://github.com/cfpb/cfgov-refresh/pull/3063/commits/7300ea4e98231afe4a6ccfb538242ac2c040656d

File: cfgov/unprocessed/css/blog-docs.less (hunk 2 above)
Comment (anselmbradford) [inline cfgov/unprocessed/css/blog-docs.less]: Fixed in https://github.com/cfpb/cfgov-refresh/pull/3063/commits/7300ea4e98231afe4a6ccfb538242ac2c040656d

**Comment by sebworks** [comment]
Great stuff man, 💯

**Pull request merged**